src/components/10-atoms/input-text/README.md

```diff
@@ -146,6 +146,16 @@ The string-valued `info` provides the label text as HTML.
 
 _Note: The application is responsible for sanitizing HTML!_
 
+### pattern
+
+It is a string value. Useful to open up numeric keyboard on touch devices. See official HTML documentation for details.
+
+Default: ".\*" This RegEx allows everything. An empty string causes errors on form validation.
+
+### inputmode
```

Comment:
Why didn't you use the attribute name "type"? We once decided to go with as close to the html spec as possible.

Reply:
type restricts the input a user will do. The requirement do not want to ristrict input, just want to show number keyboard. And this is done with "pattern" (iOS) and "inputmode" (all others).